tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -20,14 +18,15 @@ def get_pytorch_decoder(model, input_shape, example_inputs, args):
         log.error("PyTorch frontend loading failed")
         raise e
     try:
-        import nncf
-        from nncf.torch.nncf_network import NNCFNetwork
-        from packaging import version
-
-        if isinstance(model, NNCFNetwork):
-            if version.parse(nncf.__version__) < version.parse("2.6"):
-                raise RuntimeError(
-                    "NNCF models produced by nncf<2.6 are not supported directly. Please export to ONNX first.")
+        if 'nncf' in sys.modules:
```

Comment:
I didn't get it. How does it help to suppress confusing logs?
We get confusing logs during `import nncf` and check `if 'nncf' in sys.modules` will pass.